other/upgrade.php

```diff
@@ -3252,6 +3252,10 @@ function ConvertUtf8()
 		db_extend();
 		$queryTables = $smcFunc['db_list_tables'](false, $db_prefix . '%');
 
+		$queryTables = array_values(array_filter($queryTables, function($v){
+			return !(stripos($v, 'backup_') === 0);
```

Comment:
```suggestion
			return stripos($v, 'backup_') !== 0;
```

just a silly change

Reply:
Well that could mean it would match `my_backup_table`.  Instead of just matching `backup_table`.  That was my intention at least.

Reply:
No, they are equivalent statements.

Reply:
oh yea, its still strict type checking.